starknet_py/net/client_models.py

```diff
@@ -950,6 +951,7 @@ class L1HandlerTransactionTrace:
     Dataclass representing a transaction trace of an L1_HANDLER transaction.
     """
 
+    execution_resources: Optional[ExecutionResources]
```

Comment:
@DelevoXDG please let's make sure we have this field in jvm/swift as well, as per latest discussion on swm-sw channel this is a spec bug